loop/config.js

```diff
@@ -540,6 +554,11 @@ if (conf.get('fxaOAuth').activated && conf.get('fxaOAuth').client_id === "") {
     "Set fxaOAuth.activated config key to false to continue";
 }
 
+if (conf.get('env') === 'test') {
+  process.env.AWS_ACCESS_KEY_ID = 'TESTME';
```

Comment:
I don't understand why these are here? We cannot pass the configuration variables to s3 directly?
In any case, a comment explaining why we are setting env variables here would help to understand :-)